### DISCUSSION TITLE
Autocomplete in rust document

### DISCUSSION THREAD

**Discussion by marti1125** [zed-industries/zed#6568] [Feature Requests]
I tried to get suggestion just pressing crtl + space in mac os but doesn't work?
is it possible to use rust-analyzer?
Thanks!

**Comment by JosephTLyons** [comment]
@marti1125 - `rust-analyzer` is what Zed is using.  Do you have a full Rust project open in Zed, or just a single file?  I don't think we get language server features when its just a single Rust file.

**Reply by marti1125** [reply]
With single files and cargo project didn't work.

**Reply by marti1125** [reply]
it works with "enable_language_server": true,

**Comment by marti1125** [comment]
Zed 0.91.3 for Mac OS
I notices my settings does't has "enable_language_server": true,

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
